frontend/poll-app/src/pages/FindAvailablePoll.jsx

```diff
@@ -102,28 +95,24 @@ function FindAvailablePoll() {
     }
 
     useEffect(() => {
-        fetchAnsweredPolls(username);
+        fetchAnsweredPolls();
     }, []);
 
-
-
     const handleSubmit = (e) => {
         e.preventDefault(); // Prevent default form submission behavior
-        fetchPollDetails(pollId)
+        getPollRequest(pollId)
             .then(data => {
-                if (data && !data._id) {
+                if (!data._id) {
```

Comment:
This might be causing "Poll not found" to be alerted for any poll id that gets entered because the response no longer contains an _id field.

Steps to reproduce: Go to /vote and enter in 661391f8fba77c303c1bc4d9 (sample_user's most recent quick poll)

Reply:
Is this a type issue? `data` here is a Response which hasn't been read to json. Chaining a `.then(res => res.json())` after making the request would probably resolve this